Makefile

```diff
@@ -0,0 +1,87 @@
+MODULES += nginx
+MODULES += keycloak
+MODULES += hedgedoc
+MODULES += grafana
+MODULES += prometheus
+MODULES += mastodon
+#MODULES += pixelfed
+
+include env.production
+domain_name := $(DOMAIN_NAME)
+
+
+help:
+	@echo "usage: make run"
+UC = $(shell echo '$1' | tr '[:lower:]' '[:upper:]')
+
+DOCKER = \
+	$(foreach m,$(MODULES),. data/$m/secrets && ) \
+	docker-compose \
+		--env-file env.production \
+		$(foreach m,$(MODULES),--file ./$m.yaml) \
```

Comment:
If this gets merged, I might want to extend it to support podman/podman-compose.